src/python/pants/backend/python/dependency_inference/scripts/dependency_parser.py

```diff
@@ -79,6 +87,12 @@ def find_token(string):
             if alias.asname and token[1] != alias.asname:
                 find_token(alias.asname)
 
+    def maybe_add_string_dependency(self, node, s):
+        if STRING_IMPORTS and STRING_IMPORT_REGEX.match(s):
+            self.weak_imports.setdefault(s, node.lineno)
+        if ASSETS and ASSET_REGEX.match(s):
+            self.assets.add((None, s))
```

Comment:
This is always set to `None` for now, but I think you intend to eventually infer the `pkg_name`? I recommend deleting that part of the code for now. This script is an implementation detail and we can add the pkg_name code whenever. I think this will simplify callers a bit.